### PR TITLE
Get range method

### PR DESCRIPTION
**Pull request by mmsc2**
this PR is required for this one to work:
https://github.com/lambdaclass/cairo-rs-py/pull/88
since it implements a function that is used to map between rust types to python types.

### CODE REVIEW

**Review by fmoletta — CHANGES_REQUESTED** [review]
Please add tests for both methods

File: src/vm/vm_core.rs [hunk 1]
```diff
@@ -738,6 +738,15 @@ impl VirtualMachine {
         self.memory.get_range(addr, size)
     }
 
+    ///Gets n elements from memory starting from addr (n being size)
+    pub fn get_continuos_range(
+        &self,
+        addr: &MaybeRelocatable,
+        size: usize,
+    ) -> Result<Vec<MaybeRelocatable>, MemoryError> {
+        self.memory.get_continuos_range(addr, size)
```
Comment (Juan-M-V) [inline src/vm/vm_core.rs:747]: it should be continuous instead

Comment (mmsc2) [reply src/vm/vm_core.rs]: fixed

File: src/vm/vm_memory/memory.rs [hunk 2]
```diff
@@ -237,6 +237,23 @@ impl Memory {
         Ok(values)
     }
 
+    pub fn get_continuous_range(
+        &self,
+        addr: &MaybeRelocatable,
+        size: usize,
+    ) -> Result<Vec<MaybeRelocatable>, MemoryError> {
+        let mut values = Vec::new();
```
Comment (fmoletta) [inline src/vm/vm_memory/memory.rs:245]:
```suggestion
        let mut values = Vec::with_capacity(size);
```

Comment (mmsc2) [reply src/vm/vm_memory/memory.rs]: Targeted

File: src/vm/errors/memory_errors.rs [hunk 3]
```diff
@@ -34,6 +34,8 @@ pub enum MemoryError {
     MissingSegmentUsedSizes,
     #[error("Segment at index {0} either doesn't exist or is not finalized.")]
     SegmentNotFinalized(usize),
+    #[error("Found a memory gap when calling get_continous_range")]
```
[the remark this answers is not in the document]
Comment (mmsc2) [reply src/vm/errors/memory_errors.rs]: fixed

File: src/vm/vm_memory/memory.rs [hunk 4]
```diff
@@ -719,4 +736,67 @@ mod memory_tests {
...
+            Some(Cow::Borrowed(&value3)),
+        ];
+        assert_eq!(
+            memory.get_range(&MaybeRelocatable::from((1, 0)), 4),
+            Ok(expected_vec)
+        );
+    }
+
+    #[test]
+    fn getcontinuous_range_for_continuous_memory() {
```
Comment (fmoletta) [inline src/vm/vm_memory/memory.rs:779]:
```suggestion
    fn get_continuous_range_for_continuous_memory() {
```

**Review by fmoletta — APPROVED** [review]
Great Work!

**Pull request merged**